streamwebs_frontend/streamwebs/forms.py

```diff
@@ -260,3 +260,18 @@ class StatisticsForm(forms.Form):
         widget=forms.DateInput(attrs={'class': 'datepicker'}),
         label=_('ending on'), required=False
     )
+
+
+class AdminPromotionForm(forms.Form):
```

Comment:
I think adding a search bar to improve functionality can also be addressed in a separate issue (probably slipped in with the rest of the changes for #266). It'd be good to merge this now, so that we can at least demo some admin permissions to Renee next week.